### ISSUE TITLE
React Fiber overview

### ISSUE THREAD

**Issue by mmdonaldson** [reactsydney/talks#44]
## Talk Date
Whenever is next free

## Talk Title
React Fiber overview

## Length
20-30min

## Twitter-sized Overview (140 characters)
Introduction to React Fiber and how it will effect us

## Detailed Overview
To come

**Comment by mmdonaldson** [comment]
Assuming this hasn't been done yet?

**Comment by joshhunt** [comment]
wow yes please! Would love to see what this means for us practically speaking and some of the benefits we get now and in the future!

**Comment by veebuv** [comment]
Super keen for this!

**Comment by jesstelford** [comment]
The community has spoken; this is an excellent talk idea!

@mmdonaldson the next meetup is May 1st, or June 5th after that - which would you prefer?

**Comment by mmdonaldson** [comment]
@jesstelford let's lock in May 1. The audience is pretty 50/50 on beginner vs good react knowledge right?

**Comment by veebuv** [comment]
@mmdonaldson from what @jesstelford mentioned before, yes, semi-split audience!

Also - https://goo.gl/kn7HD8

**Comment by jesstelford** [comment]
@mmdonaldson Yeah, it's a wide range in the audience; we have some absolute beginners, through to experts. It's up to you which audience you focus on, but can be beneficial to add explainer pieces for some of the trickier concepts (even old-timers can learn new tricks!)

Looking forward to this one :+1:

**Comment by jesstelford** [comment]
@mmdonaldson All prepared for the meetup tonight?

**Comment by mmdonaldson** [comment]
Yep but i didn't manage to get on the event list... @jesstelford

**Comment by jesstelford** [comment]
Haha, you get a free ticket ;)

**Comment by jgrancher** [comment]
@mmdonaldson Couldn't make it last night. Do you have the slides or a video somewhere?

**Comment by mmdonaldson** [comment]
@jgrancher My slides are pretty slim on content, i'll had a few notes and upload at the weekend

**Comment by jesstelford** [comment]
The video will be uploaded in the coming weeks 👍
